docs/spec/staking/state.md

```diff
@@ -2,46 +2,46 @@
 
 ### Pool
 
-The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, moving Atom
-inflation information, etc.
+The pool dynamically tracks information about the total amounts of staking
+tokens and their states (bonded or loose). Note that `NotBondedTokens` _does_
+include both `unbonding` tokens, and fully `unbonded` tokens. 
 
  - Pool: `0x01 -> amino(pool)`
```

Comment:
So is this the store key bytes? Would love some more explanation here.

Reply:
see the spec spec section `### Notation for key-value mapping`